apps/req.c

```diff
@@ -1675,7 +1652,6 @@ static int do_x509_req_init(X509_REQ *x, STACK_OF(OPENSSL_STRING) *opts)
         char *opt = sk_OPENSSL_STRING_value(opts, i);
         if (x509_req_ctrl_string(x, opt) <= 0) {
             BIO_printf(bio_err, "parameter error \"%s\"\n", opt);
-            ERR_print_errors(bio_err);
             return 0;
```

Comment:
I'm not entirely sure I understand why this app gets 'return 0' directoy instead of the 'goto end' with `ERR_print_errors()` that we see in some of the others.

Reply:
See above